src/Service.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace PodPoint\Reviews;
+
+interface Service
```

Comment:
this approach does not take into consideration we can have product and merchant reviews for reviews.co.uk

Reply:
Could we have about about this one?